lib/webpacker/manifest.rb

```diff
@@ -18,13 +18,29 @@ def refresh
     @data = load
   end
 
-  def lookup(name)
+  def lookup(name, pack_type = {})
     compile if compiling?
-    find name
+
+    # When using SplitChunks or RuntimeChunks the manifest hash will contain
+    # an extra object called "entrypoints". When the entrypoints key is not
+    # present in the manifest, or the name is not found in the entrypoints hash,
+    # it will raise a NoMethodError. If this happens, we should try to lookup
+    # a single instance of the pack based on the given name.
+    begin
+      manifest_pack_type = manifest_type(pack_type[:type])
+      manifest_pack_name = manifest_name(name, manifest_pack_type)
+
+      # Lookup the pack in the entrypoints of the manifest
+      find("entrypoints")[manifest_pack_name][manifest_pack_type]
```

Comment:
Perhaps, we can use `dig` here then? https://ruby-doc.org/core-2.3.0_preview1/Hash.html#method-i-dig